packages/strapi-parts/src/Select/Select.stories.mdx

```diff
@@ -27,7 +29,12 @@ Selects are fields that displays to users a list of options via a PopOver compon
 ## Imports
 
 ```js
-import { Select, Option } from '@strapi/parts/Select';
+import { Select } from '@strapi/parts/Select';
```

Comment:
`import { Select, Option } from '@strapi/parts/Select';`

The option is exported by the Select.